projects/ngx-prx-styleguide/src/public_api.ts

```diff
@@ -37,39 +55,81 @@ export { HalHttpError } from './lib/hal/remote/halremote';
 export { MockHalService } from './lib/hal/mock/mock-hal.service';
 export { MockHalDoc } from './lib/hal/mock/mock-haldoc';
 
+export * from './lib/header/header.component';
+export * from './lib/header/navitem.component';
+export * from './lib/header/navuser.component';
 export { HeaderModule } from './lib/header/header.module';
 
+export { HeroComponent } from './lib/hero/hero.component';
 export { HeroModule } from './lib/hero/hero.module';
 
+export { IconComponent } from './lib/icon/icon.component';
 export { IconModule } from './lib/icon/icon.module';
 
+export { ImageLoaderComponent } from './lib/image/image-loader.component';
 export { ImageModule } from './lib/image/image.module';
 
+export { ModalComponent } from './lib/modal/modal.component';
 export { ModalModule } from './lib/modal/modal.module';
 export { ModalService } from './lib/modal/modal.service';
 
+export { PagingComponent } from './lib/paging/paging.component';
 export { PagingModule } from './lib/paging/paging.module';
 
+export { SelectComponent } from './lib/select/select.component';
 export { SelectModule } from './lib/select/select.module';
 
+export { SpinnerComponent } from './lib/spinner/spinner.component';
 export { SpinnerModule } from './lib/spinner/spinner.module';
 
-export { StickyModule } from './lib/sticky/sticky.module';
-
+export * from './lib/status-bar/status-bar-icon/status-bar-icon.component';
+export * from './lib/status-bar/status-bar-image/status-bar-image.component';
+export * from './lib/status-bar/status-bar-link/status-bar-link.component';
+export * from './lib/status-bar/status-bar-text/status-bar-text.component';
+export * from './lib/status-bar/status-bar.component';
 export { StatusBarModule } from './lib/status-bar/status-bar.module';
 
+export { StickyDirective } from './lib/sticky/sticky.directive';
+export { StickyModule } from './lib/sticky/sticky.module';
+export { StickyService } from './lib/sticky/sticky.service';
+
+export { TabComponent } from './lib/tab/tab.component';
 export { TabModule } from './lib/tab/tab.module';
 export { TabService } from './lib/tab/tab.service';
 
+export { TagsComponent } from './lib/tags/tags.component';
 export { TagsModule } from './lib/tags/tags.module';
 
+export { ToastrComponent } from './lib/toastr/toastr.component';
 export { ToastrModule } from './lib/toastr/toastr.module';
 export { ToastrService } from './lib/toastr/toastr.service';
 
-
-export { UploadModule } from './lib/upload/upload.module';
 export { PlayerService } from './lib/audio';
-export { MimeTypeService, UploadService, Upload } from './lib/upload/service';
-export { AudioFileModel, AudioVersionModel, AudioVersionTemplateModel, AudioFileTemplateModel } from './lib/upload/model';
-export { UploadableModel, HasUpload, createGetUploads, createSetUploads } from './lib/upload/model/upload';
-export { DurationPipe, FileSelectDirective, FileSizePipe } from './lib/upload/file';
+export * from './lib/upload/file/duration.pipe';
```

Comment:
hate to say it after how hard it was for Peter to move this all over here, but this should probably be moved back to publish if some kind of resuable uploader that doesn't use BaseModel can't be extracted out of it